tutorials/pom.xml

```diff
@@ -0,0 +1,93 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I think this will be deleted if the pipeline is running through

Reply:
I guess tutorials/index.asciidoc also, because of scripts/copyPomFiles.sh